src/core/SampleBuffer.cpp

```diff
@@ -296,72 +296,41 @@ void SampleBuffer::update( bool _keep_settings )
 }
 
 
-void SampleBuffer::convertIntToFloat ( int_sample_t * & _ibuf, f_cnt_t _frames, int _channels)
+void SampleBuffer::convertIntToFloat (int_sample_t * & _ibuf, f_cnt_t _frames, int _channels)
```

Comment:
Not necessary to change, but since we touched this we should remove the `_` prefix as per new coding conventions.
```suggestion
void SampleBuffer::convertIntToFloat(int_sample_t * & ibuf, f_cnt_t frames, int channels)
```

Reply:
There's a bunch of these underscored variables in `visualise()` too, do I get rid of those as well?

Reply:
I think it's up to you, it can be a bit of a rabbit hole. It's always welcome that those code style issues are fixed, but since it's outside the scope of the PR it's alright if you want to leave it for another one as far as getting this one approved goes! :smile:

Reply:
If you touched the line, that's where I suggested it.